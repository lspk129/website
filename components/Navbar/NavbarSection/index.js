import styled from 'styled-components';

const NavbarSection = styled.nav`
  width: 100%;
  background-color: ${props => props.theme.main};
  overflow: hidden;
  display: grid;
  grid-template-rows: 1fr;
  z-index: 10;
`;

export default NavbarSection;
