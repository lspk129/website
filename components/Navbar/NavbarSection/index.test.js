import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import NavbarSection from './';

describe('NavbarSection', () => {
  const shallowWrapper = shallow(<NavbarSection />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<NavbarSection />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders nav element', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper).toHaveTagName('nav');
  });
});
