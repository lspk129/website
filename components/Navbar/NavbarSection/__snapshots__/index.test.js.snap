// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarSection renders correctly 1`] = `
<nav
  className="sc-bdVaJa joTcTr"
/>
`;
